### ISSUE TITLE
UAST viewer does not render nodes when UAST arg is a string

### ISSUE THREAD

**Issue by carlosms** [src-d/gitbase-web#402]
For the following query:
```sql
select 'console.log("hello world");
console.log("bye");' as code, UAST('console.log("hello world");
console.log("bye");', 'javascript') as uast
```

The UAST button opens a panel where the body `Nodes` cannot be expanded. Clicking on `+` does nothing.
![image](https://user-images.githubusercontent.com/1469173/55964844-5512d900-5c6d-11e9-9749-566a86180119.png)

**Comment by smacker** [comment]
Just a note: it must be gitbase-web bug not the uast-viewer itself.
Because `console.log("hello world");\nconsole.log("bye");` is displayed correctly on bblfsh playground.

**Comment by se7entyse7en** [comment]
This is issue affects also non-string argument to `UAST` but is much extensive. It can be replicated also with the `Files named main.go` example and affects also other types of elements such as `Doc` and `Path`.

**Comment by se7entyse7en** [comment]
The expansion of the node works well in the code viewer modal, but doesn't in the uast viewer one. By using the react dev tool on chrome, I noticed that the props `onNode{Click,Hover,Toggle}` are not passed in the `FlatUASTViewer` used in the uast viewer modal.

I found that these props are passed [here](https://github.com/bblfsh/uast-viewer/blob/v0.4.0/src/withUASTEditor.js#L190) when using the `withUASTEditor`, and if I understand correctly these are the props that make the collapsible nodes work.

IMHO the behavior that we have in the code viewer given by the aforementioned props could be the default. I mean that I don't see the need for customizing those. What I'd do is to move those handlers into `FlatUASTViewer` and to change the `onNode{Click,Hover,Toggle}` into `enableNode{Click,Hover,Toggle}`.

WDYT?
